seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -310,7 +409,8 @@
         "<blockquote>.blue-text {<br>&nbsp;&nbsp;color: blue;<br>}</blockquote>",
```

Comment:
Okay, so I'm replacing all of the existing description with your text - agree it's repetitive. Is it okay if I leave the translations alone? Google Translate has its limitations (especially with the Russian one)...